### DISCUSSION TITLE
Cannot reverify Education Dev Pack.

### DISCUSSION THREAD

**Discussion by Butanediol** [community/community#49135] [GitHub Education]
I know this might be duplicated. But other people seems to have this automatically resolved after a period of time. Mine doesn't.

I uploaded my school id along with translation but it still yells me as below:

> Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I tried to add my full name to the bio but no help. And i issued a ttcket over a week ago and still get no respond.

Please help me pass this.

**Comment by jge162** [comment]
If you already opened a support ticket then you will have to wait a bit for assistance. Sounds like you made all the correct changes.

**Reply by Butanediol** [reply]
Good news. They finally responded to my tickets.
Bad news, they just copy and paste GitHub support page and closed it.
Why is GitHub so bad at supporting?

**Comment by xcdh-233** [comment]
好兄弟 ，
你解决问题了吗
曹

**Reply by jge162** [reply]
do not understand, sorry

**Reply by xcdh-233** [reply]
No problem
we are 伐木累

**Reply by Butanediol** [reply]
Unfortunately no.
I’ll just wait to see when will GitHub reply to my ticket.

没解决，我就等 GitHub 回我工单了。

**Reply by xcdh-233** [reply]
我已经 wait 了 一 week了
nobody 回复 my 问题

**Reply by xcdh-233** [reply]
我甚至 在 学信网  spend 30RMB 去 get 了 一份  英文 report
but 毫无luan用

**Reply by samus-aran** [reply]
Hi @Butanediol & @xcdh-233 👋 
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Reply by SiweiPro** [reply]
six six six

**Comment by samus-aran** [comment]
Hi @Butanediol 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. Ensure this translation includes your name, school name etc.

Hope this helps. Let us know how you get on 👍

**Reply by Butanediol** [reply]
I've tried uploading a document with translation. And changed github profile to my real name, school, etc. It still doesn't work.
Does GitHub have manual check?
